### PR TITLE
Add new icon, and used in header component

### PR DESCRIPTION
**Pull request by wouter1975**
* Add new icon, and used in header component.
* Placement in front of breadcrumb still te be done

### CODE REVIEW

File: packages/orchestrator-ui-components/src/icons/WfoSideMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    color = '#000000',
+}) => (
+    <svg
+        width={width}
+        height={height}
+        viewBox="0 0 60 60"
+        version="1.1"
+        xmlns="http://www.w3.org/2000/svg"
+    >
+        <title>icon/logout</title>
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/icons/WfoSideMenu.tsx:17]: this should be icon/sidemenu

File: packages/orchestrator-ui-components/src/icons/WfoSideMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        <g id="Symbols" stroke="none" strokeWidth="6" fill="none">
+            <g
+                id="icon/logout"
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/icons/WfoSideMenu.tsx:20]: this should be icon/sidemenu

**Review by DutchBen — APPROVED** [review]

File: packages/orchestrator-ui-components/src/icons/WfoSideMenu.tsx (hunk 2 above)
Comment (wouter1975) [reply packages/orchestrator-ui-components/src/icons/WfoSideMenu.tsx:20]: check

File: packages/orchestrator-ui-components/src/icons/WfoSideMenu.tsx (hunk 1 above)
Comment (wouter1975) [reply packages/orchestrator-ui-components/src/icons/WfoSideMenu.tsx:17]: check

File: packages/orchestrator-ui-components/src/icons/WfoSideMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import React, { FC } from 'react';
```
[the remark this answers is not in the document]
Comment (wouter1975) [reply packages/orchestrator-ui-components/src/icons/WfoSideMenu.tsx:1]: added!

File: packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoPageHeader/WfoPageHeader.tsx [hunk 4]
```diff
@@ -51,10 +51,10 @@ export const WfoPageHeader: FC<WfoPageHeaderProps> = ({
                 </EuiHeaderSectionItem>
 
                 <EuiButtonIcon
-                    aria-label="Logout"
+                    aria-label="Show/Hide side menu"
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoPageHeader/WfoPageHeader.tsx:54]: Do we want translations for aria labels as well?

**Review by ricardovdheijden — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoPageHeader/WfoPageHeader.tsx (hunk 4 above)
Comment (wouter1975) [reply packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoPageHeader/WfoPageHeader.tsx:54]: Don't think this is necessary.

**Pull request merged**